src/Debugger-Tests/DebuggerTest.class.st

```diff
@@ -34,9 +34,9 @@ DebuggerTest >> setUp [
 
 { #category : #utilities }
 DebuggerTest >> settingUpSessionAndProcessAndContextForBlock: aBlock [
-	context := aBlock asContext.
-	process := Process 
-	    forContext: context 
-	    priority: Processor userInterruptPriority.
+	process := aBlock newProcess.
+	[process step closure == aBlock] whileFalse.
+	process step.
```

Comment:
The separate step here is to really activate the block - to set the context pc to the next expected instruction. So the following stepInto will do exactly what would be expected in the debugger.
It allowed to remove from many tests the first stepInto to prepare the process block.